build-tools/packages/build-tools/src/fluidBuild/buildGraph.ts

```diff
@@ -532,11 +533,14 @@ export class BuildGraph {
 		return succeeded;
 	}
 
-	public async build(timer?: Timer): Promise<BuildResult> {
+	public async build(timer?: Timer, spinner?: Spinner): Promise<BuildResult> {
```

Comment:
Does it need to be passed as a parameter? In this case the caller starts and finishes its use of it so it seems like it wouldn't cause issues, but in general it feels like this would be better off being its own instance so it doesn't potentially interfere with the one passed in (if that one was still in progress, for example)?